deno.jsonc

```
@@ -24,24 +24,18 @@
   },
   "include": ["."],
   "tasks": {
-    "download_frame_metadata": "deno run -A --no-check=remote _tasks/download_frame_metadata.ts",
-    "build_wasm": "deno run --unstable -A _tasks/build_wasm.ts --out",
-    "build_hashers_wasm": "deno task build_wasm bindings/hashers --features hashers",
-    "build_sr25519_wasm": "deno task build_wasm bindings/sr25519 --features sr25519",
-    "build_ss58_wasm": "deno task build_wasm bindings/ss58 --features ss58",
-    "build_all_wasm": "deno task build_hashers_wasm && deno task build_sr25519_wasm && deno task build_ss58_wasm",
-    "star": "deno run -A --no-check=remote _tasks/star.ts && deno cache --no-check=remote _star.ts",
-    "build_npm_pkg": "deno run -A --no-check=remote _tasks/build_npm_pkg.ts",
+    "run": "deno run -A --no-check=remote",
+    "download_frame_metadata": "deno task run _tasks/download_frame_metadata.ts",
+    "build:_wasm": "deno task run --unstable _tasks/build_wasm.ts --out",
+    "build:hashers_wasm": "deno task build:_wasm bindings/hashers --features hashers",
+    "build:sr25519_wasm": "deno task build:_wasm bindings/sr25519 --features sr25519",
+    "build:ss58_wasm": "deno task build:_wasm bindings/ss58 --features ss58",
+    "build:wasm": "deno task build:hashers_wasm && deno task build:sr25519_wasm && deno task build:ss58_wasm",
+    "star": "deno task run _tasks/star.ts && deno cache --no-check=remote _star.ts",
+    "build_npm_pkg": "deno task run _tasks/build_npm_pkg.ts",
     "lock": "deno task star --lock=lock.json --lock-write",
-    "example:metadata": "deno run -A --no-check examples/metadata.ts",
-    "example:balance": "deno run -A --no-check=remote examples/balance.ts",
-    "example:playground": "deno run -A --no-check examples/playground.ts",
-    "example:transfer": "deno run -A --no-check examples/transfer.ts",
-    "example:events": "deno run -A --no-check examples/events.ts",
-    "example:rpc_call": "deno run -A --no-check examples/rpc_call.ts",
-    "example:first_ten_keys": "deno run -A --no-check examples/first_ten_keys.ts",
     "test": "deno test -A --no-check=remote -L=info",
-    "test-update": "deno task test -- -- --update",
+    "test_update": "deno task test -- -- --update",
     "update_snapshots": "deno test -A --no-check=remote -L=info -- --update",
```

Whoops, these are duplicative.
```suggestion
    "test_update": "deno task test -- -- --update",
```

---

Also, should `test_update` be `test:update`?